mandatory/4-tax.js

```diff
@@ -17,7 +19,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(prices) {
+  let afterTax = prices + calculateSalesTax(prices);
+  return `£${afterTax}`
+}
```

Comment:
you might need to use .toFixed(2) to force it to give two decimals

Reply:
Yeah that is a good idea. But , this passed the test too. I guess it is because we've multiplied it by 0.2 . I will look into it. Thanks !